crates/context/config/src/client/env/proxy/query/context_storage_entries.rs

```diff
@@ -111,7 +112,12 @@ impl Method<Stellar> for ContextStorageEntriesRequest {
     const METHOD: &'static str = "context_storage_entries";
 
     fn encode(self) -> eyre::Result<Vec<u8>> {
-        todo!()
+        let mut encoded = Vec::new();
+        encoded.extend_from_slice(&self.offset.to_le_bytes());
+
+        encoded.extend_from_slice(&self.limit.to_le_bytes());
```

Comment:
don't you need to xdr encode these? do they turn out the same as manually encoding them?